### PR TITLE
Twitter

### PR DESCRIPTION
**Pull request by Diana20920**
### Completed
- [x] Twitter API call for most recent tweets related to `stocks`
- [x] Tests for POROs, Facades, and Services (Finnhub + Twitter)
- [x] Endpoint tested and available locally for FE `http://localhost:3000/api/v1/tweets`
- [x] Endpoint tested and available locally for FE `http://localhost:3000/api/v1/news`

#### Applied Changes
- From `faraday.headers['authorization'] = ENV['twitter_token']` to `faraday.headers['Authorization'] = "Bearer #{ENV['twitter_token']}"`
- Tweets retrieved are now a total of 3 instead of 10.

**Endpoints are 100% functional on local**
**Coverage is at `94.23%` due to `app/models/application_record.rb`**

### CODE REVIEW

File: app/services/tweet_service.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
+class TweetService
+  def self.conn
+    Faraday.new(url: 'https://api.twitter.com') do |faraday|
+      faraday.headers['authorization'] = ENV['twitter_token']
```
Comment (ghost) [inline app/services/tweet_service.rb]: This line needs to be:
faraday.headers['Authorization'] = "Bearer #{ENV['twitter_token']}"
to authorize with twitter

File: app/facades/tweet_facade.rb [hunk 2]
```diff
@@ -0,0 +1,7 @@
+class TweetFacade
+  def self.recent_tweets
+    TweetService.recent_tweets.map do |data| 
+        Tweet.new(data)
+    end.first(10)
```
Comment (ghost) [inline app/facades/tweet_facade.rb]: Let's limit this to three

**Review by ghost — COMMENTED** [review]
Ya'll crushed it yesterday

File: app/services/tweet_service.rb (hunk 1 above)
Comment (Diana20920) [reply app/services/tweet_service.rb]: That makes so much more sense that what I had @brueck1988 do. TY! Alexander, make sure you reflect the change on your local `application.yml` but let me know if you're not sure how 😃

**Review by ghost — APPROVED** [review]

**Pull request merged**